packages/inventory-insights/src/Insights.js

```diff
@@ -135,11 +135,14 @@ class InventoryRuleList extends Component {
                 return rowKey % 2 === 0 && rowVal.rule.rule_id === rule.rule_id && rowVal.selected;
             });
             selected = selected.length ? true : false;
+            const isOpen = rows.filter((rowVal, rowKey) => {
```

Comment:
so we do this same thing on  line 134, the map through rows... smells like an opportunity to consolidate the.... 🤔